code/modules/awaymissions/mission_code/ruins/whiteship.dm

```diff
@@ -0,0 +1,8 @@
+// MARK: LORE CONSOLES
+
+/obj/machinery/computer/loreconsole/whiteship
+	entries = list(
+		new/datum/lore_console_entry(
+			"DON'T FORGET AGAIN.",
```

Comment:
https://www.youtube.com/watch?v=YLeid-bIRQA
```suggestion
			"DON'T FORGET.",
```

Reply:
https://youtu.be/55RnOgnv5Dc?feature=shared